README.md

```diff
@@ -1 +1,26 @@
-# Portfolio
\ No newline at end of file
+# My Portfolio
+
+> This is my portfolio describing what I am capable of doing and what value I can bring in a team.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://htmlpreview.github.io/?https://github.com/samWfelice/Portfolio/blob/mobile-skeleton/index.html)
+
+
+## Authors
+
+👤 **Samuel Fleice**
+
+- GitHub: [@samWfelcie](https://github.com/samWfelice)
+- Twitter: [@samwfelice](https://twitter.com/samwfelice)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/samfelice/)
+
+## Show your support
+
+Give a ⭐️ if you like this project!

```

Comment:
- [ ] **MANDATORY REQUIREMENTS**
 Great job on implementing the changes required by the previous reviewer. Though they still missed pointing out this also, please do consider this so you improve your readme to be more professional in your next project milestone.
 - Your readme is missing some headers which are mandatory. **[Acknowledgement, License, Getting started, and Contributing]** Do try adding them following this [microverse readme template](https://github.com/microverseinc/readme-template).